packages/core/DatadogSDKReactNative.podspec

```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license      = package["license"]
   s.authors      = package["author"]
 
-  s.platforms    = { :ios => "11.0", :tvos => "11.0" }
+  s.platforms    = { :ios => "13.4", :tvos => "11.0" }
```

Comment:
This was due to the bitrise build failing [with an error about the deployment target version](https://app.bitrise.io/build/b5687b19-ee10-4f76-8bc8-03d42b161d52)

Reply:
Where this requirement (to have `13.4`) is coming from? Is it from RN 0.73? Anyway, such change is breaking in terms of backward compatibility, so I think it is better to discuss with iOS team how to keep this version aligned with iOS SDK or, if it is coming from RN 0.73, to have this only in case of using with RN 0.73.

Reply:
You can change the target like mentioned in [this comment](https://github.com/DataDog/dd-sdk-reactnative/issues/536#issuecomment-1746691907) or change the target of the app.
Xcode 15 does not support iOS 11, so we can bump this once Xcode 14 gets deprecated.